package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nodejs-dashboard",
-  "version": "0.4.1",
+  "version": "0.5.0",
```

Comment:
Again, typically we don't bump versions in PRs. Also, this differs from changelog entry.